docs/user-guide/managing-deployments.md

```diff
@@ -57,6 +57,49 @@ deployment "my-nginx" deleted
 service "my-nginx-svc" deleted
 ```
 
+If you happen to organize your resources across several subdirectories within a particular directory, you can recursively perform the operations on the subdirectories also, by specifying `--recursive` or `-R` alongside the `--filename,-f` flag.
```

Comment:
Looking at this doc, I think you should move your changes after the --selector example. I believe it's more common to try a bulk operation based on a label selector rather than a recursive one.